app/code/community/ProxiBlue/OrderSyncQueRunner/etc/api.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<config>
+    <api>
+        <resources>
+            <order_sync_que_runner translate="title" module="ordersyncquerunner">
+                <model>ordersyncquerunner_resource/que_api</model>
+                <title>Order Sync Que Runner</title>
+                <acl>order_sync_que_runner</acl>
+                <methods>
+                    <list translate="title" module="ordersyncquerunner">
+                        <title>List New Order Increment Ids</title>
+                        <method>listNewOrdersQue</method>
```

Comment:
Might be a good idea to start using the correct spelling of queue for the API ^^